uber/automated_emails.py

```diff
@@ -121,23 +121,21 @@
            ident='volunteer_again_inquiry')
 
 StopsEmail('{EVENT_NAME} Volunteer Badge Confirmation', 'placeholders/volunteer.txt',
-           lambda a: a.placeholder and a.first_name and a.last_name
-                                      and a.registered_local > c.PREREG_OPEN,
+           lambda a: a.placeholder and a.registered_local > c.PREREG_OPEN,
```

Comment:
Do we want this to include also test for `a.staffing`? I know it didn't before, but it might be a little weird for someone to get a volunteer email if they have no intention of volunteering.

Reply:
That's already included because we are using the `StopsEmail` category, which always applies the lambda filter `a.staffing` on top of whatever we put here. In fact, the email above this one is checking for a.staffing twice! I'm gonna remove that because that's just silly.